### PR TITLE
Create 111111 as default files

### PR DESCRIPTION
**Pull request by YouQuan2007**

### CODE REVIEW

File: data/Course Student List/LKM400 - BAHASA MALAYSIA.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Student 1 Name	222222
```
Comment (Michelle-Lohwt) [inline data/Course Student List/LKM400 - BAHASA MALAYSIA.txt]: why got 222222

File: data/Course Student List/HFE224 - APPRECIATION OF ETHICS AND CIVILISATIONS.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Student 1 Name	222222
```
Comment (Michelle-Lohwt) [inline data/Course Student List/HFE224 - APPRECIATION OF ETHICS AND CIVILISATIONS.txt]: why got 222222

File: data/Course Student List/CPT115 - MATHEMATICAL METHODS FOR COMPUTER SCIENCE.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Student 1 Name	222222
```
Comment (Michelle-Lohwt) [inline data/Course Student List/CPT115 - MATHEMATICAL METHODS FOR COMPUTER SCIENCE.txt]: why got 222222

File: data/Course Student List/ATW202 - BUSINESS RESEARCH METHOD.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+Student 1 Name	222222
```
Comment (Michelle-Lohwt) [inline data/Course Student List/ATW202 - BUSINESS RESEARCH METHOD.txt]: why got 222222

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
why got 222222

File: data/Course Student List/LKM400 - BAHASA MALAYSIA.txt (hunk 1 above)
Comment (YouQuan2007) [reply data/Course Student List/LKM400 - BAHASA MALAYSIA.txt]: at my side its already unavailable but idk why it is pushed

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**